src/py/mat3ra/made/tools/build/defect/__init__.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from mat3ra.utils.factory import BaseFactory
+from mat3ra.made.material import Material
+
+from .builders import PointDefectBuilderParameters
+from .configuration import PointDefectConfiguration
+from .enums import PointDefectTypeEnum
+
+
+class DefectBuilderFactory(BaseFactory):
+    __class_registry__ = {
+        PointDefectTypeEnum.VACANCY: "mat3ra.made.tools.build.defect.builders.VacancyPointDefectBuilder",
```

Comment:
Does this understand local references `.builders.Vacancy...`?

Reply:
Looks like no.
```
tests/py/unit/test_tools_build_defect.py::test_create_vacancy - 
TypeError: the 'package' argument is required to perform a relative import for '.builders'
```